xlineapi/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "xlineapi"
+version = "0.1.0"
+edition = "2021"
+authors = ["DatenLord <dev@datenlord.io>"]
+description = "Xline RPC API"
+license = "Apache-2.0"
+readme = "README.md"
+repository = "https://github.com/datenlord/Xline/tree/master/xlineapi"
+categories = ["xlineapi"]
```

Comment:
The `xlineapi` is a name, not a category. How about `RPC`, `Interfaces`, or `api`?

Reply:
done